### PR TITLE
Multistore publish synchronize

### PR DESCRIPTION
**Pull request by gerner-spryker**
- Developer(s): @ehsanmx, @gerner-spryker

- Peer Reviewer: @ehsanmx 

- Ticket: https://spryker.atlassian.net/browse/CORE-2793
- Ticket: https://spryker.atlassian.net/browse/CORE-2977

- Core PR: https://github.com/spryker/spryker/pull/3132

- Release Overview: -

#### Please confirm

- [x] No new OS components - or they have been approved by legal department
- [ ] All new or heavily changed classes get an "A" rating (Scrutinizer), except Facades, Factories, QueryContainers and Tests

#### Documentation

- [x] Functional documentation provided or in progress?
- [x] Integration guide for projects provided or not needed?
- [x] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   EventBehavior              | major                |                  |

#### Release Notes

Changing spy_event_behavior_entity_change primary key type to BIGINT.

-----------------------------------------

#### Module RabbitMq

_**Mini-Major:** Added functionality with incompatible API changes_

Def of done (by responsible developer):
- [x] All methods in facade, client and plugin interfaces provide an [API-doc](https://academy.spryker.com/understanding_spryker/definition_api.html).
- [x] New code fits to the architecture rules.
- [x] All new or changed business logic is covered by functional tests (in Zed business layer).
- [x] Migration guide provided.

Improvements

- Changing spy_event_behavior_entity_change primary key type to BIGINT.

### CODE REVIEW

**Comment by ehsanmx** [comment]
maybe we can release it with version 1

**Comment by dereuromark** [comment]
the release notes should state the reason why this changed.
the Technical part should be limited to the Changelog.

File: src/Spryker/Zed/EventBehavior/Persistence/Propel/Schema/spy_event_behavior.schema.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\EventBehavior\Persistence" package="src.Orm.Zed.EventBehavior.Persistence">
 
     <table name="spy_event_behavior_entity_change">
-        <column name="id_event_behavior_entity_change" type="integer" autoIncrement="true" primaryKey="true"/>
+        <column name="id_event_behavior_entity_change" type="BIGINT" autoIncrement="true" primaryKey="true"/>
```
Comment (aurisl) [inline src/Spryker/Zed/EventBehavior/Persistence/Propel/Schema/spy_event_behavior.schema.xml]: This is probably good enough for now. But you should probably consider reseting sequence. Or have custom sequences which are not depended on data insert/delete.

**Comment by ehsanmx** [comment]
This will merge with this PR https://github.com/spryker/event-behavior/pull/6

**Pull request closed without merging**